src/App.tsx

```diff
@@ -22,14 +22,19 @@ import "@ionic/react/css/text-transformation.css";
 
 /* Theme variables */
 import { PrivateRoute } from "./auth/PrivateRoute";
+import { PublicRoute } from "./auth/PublicRoute";
 import { AuthPage } from "./pages/AuthPage";
 import { DashboardPage } from "./pages/DashboardPage";
+import { FarmChecklist } from "./pages/farmers/checklist";
+import { FarmRecordComplete } from "./pages/farmers/checklist/complete";
+import { FarmerDetailPage } from "./pages/farmers/FarmerDetailPage";
+import { FarmersListPage } from "./pages/farmers/FarmersListPage";
 import { LogDataPage } from "./pages/LogDataPage";
 import "./theme/variables.css";
-import { FarmersListPage } from "./pages/FarmersListPage";
-import { PublicRoute } from "./auth/PublicRoute";
 
-setupIonicReact();
+setupIonicReact({
+  mode: "ios"
```

Comment:
nit: doesn't make sense to use this mode since `md` (material design for android) makes more sense for small android devices

Reply:
Okay, I'll change it